ConsoleApp1ClassWork13.05/ConsoleApp1ClassWork13.05/Program.cs

```diff
@@ -0,0 +1,225 @@
+﻿//1. Add meeting - without walidation
+//0. Exit calendar
+//2. Show all meeting
+using Microsoft.VisualBasic.FileIO;
+using System;
+using System.ComponentModel.Design;
+using System.Globalization;
+using System.IO.Enumeration;
+using System.Text;
+using static System.Net.Mime.MediaTypeNames;
+
+Typle<string, int ,string, string>[] meetings = new Typle<string, int, string, string>[0];
+const string FileName = "meetings.csv";
+
+void ShowAll()
+
+{
+   
+    Console.WriteLine($"Start time ", 25);
+    Console.WriteLine($"Duration ", 25);
+    Console.WriteLine($"Room ", 25);
+    Console.WriteLine($"Name ", 25);
+    Console.WriteLine();
+    var fileContent= File.ReadAllLines(FileName);
+    foreach (var line in fileContent)
+    {
+        var meetingContent = line.Split(',');
+        Console.WriteLine($"{meetingContent[0], 20}"//date
+            +$"{meetingContent[1], 20}" //duration
+            + $"{meetingContent[2], 20}"//room
+            + $"{meetingContent[3], 20}");//name
+    }
+    
+    Console.ReadLine();
+}
+ bool GetRoom (DateTime dateTime, string roomName)
```

Comment:
If method is called GetRoom, my expectation would be that it returns a room. But here it returns bool, so more appropriate method name here may be "IsRoomExists" or "IsRoomOccupied", or "IsRoomBusy", depending on how we use this method and what meaning does it have